assets/src/amp-story-editor-blocks.js

```diff
@@ -217,7 +232,7 @@ const addAMPExtraProps = ( props, blockType, attributes ) => {
 const setBlockParent = ( props ) => {
 	const { name } = props;
 
-	if ( ! ALLOWED_CHILD_BLOCKS.includes( name ) ) {
+	/*if ( ! ALLOWED_CHILD_BLOCKS.includes( name ) ) {
```

Comment:
^ This is temporarily commented out for being able to test normally (since adding blocks doesn't work as expected.